src/controller.js

```diff
@@ -78,30 +78,70 @@ function buildData(dataset, mainRect) {
     : squarify(tree, mainRect, key);
 }
 
+function getMinMax(data, useTree) {
+  const vMax = useTree ? 1 : maxValue(data);
+  const vMin = useTree ? 0 : minValue(data, vMax);
+  return {vMin, vMax};
+}
+
+function getArea(chart, data, rtl, useTree) {
+  const {vMin, vMax} = getMinMax(data, useTree);
+  const {x, y} = chart.scales;
```

Comment:
What if the scales are not named x&y?

Reply:
Does it mean that the use can change the axes config? Of course it can, but the controller couldn't work correctly (setting for instance min or max). But if you say me that this is acceptable, I'll change it.

Reply:
I guess I thought it would not work with plugins etc, but now that we are going there, we should also support different scale id's too. At least try not to make more places to fix when custom scales are required :)

Reply:
I'm changing using the dataset meta and its x/y scale instances, therefore you can name the axes with whatever id.